scripts/utils/webpackconfig-vue-v3.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
```

Comment:
Obsolete because of ESLint configuration.

Reply:
Done.